### PR TITLE
Update and rename tourney-rules.md to tourney-rules.html

### PR DESCRIPTION
**Pull request by PoshEngineer**
Gave HTML format to the tournament rules section and gave better wording to some of the rules.

### CODE REVIEW

**Review by Pablete1234 — APPROVED** [review]

File: _pages/tourney-rules.html [hunk 1]
```diff
@@ -6,117 +6,109 @@
...
+        </div>
+    </div>
+    <div class="pb-2 mt-4 mb-2 border-bottom">
+        <h2 class="page-heading">F. Spectator Passes</h2>
+    </div>
+    <div class="row">
+        <ol>
+            <li>Players who have purchased any spectator pass are forbidden from affecting the outcome of the match in any way while they are spectating.</li>
+            <li>In the unfortunate event that this occurs, that player may have its pass removed without a refund and the match may be <b>voided</b>.</li>
+            <li>If the player is enlisted in the tournament, its team may be punished and that player will receive a <b>Tournament Ban</b>, including a <b>blacklist</b>from future events.</li>
```
Comment (dentmaged) [inline _pages/tourney-rules.html:112]:
```suggestion
            <li>If the player is enlisted in the tournament, its team may be punished and that player will receive a <b>Tournament Ban</b>, including a <b>blacklist</b> from future events.</li>
```

File: _pages/tourney-rules.html [hunk 2]
```diff
@@ -6,117 +6,109 @@
 ---
 
 <div class="container">
```
Comment (dentmaged) [inline _pages/tourney-rules.html:8]:
```suggestion
<div class="container pt-3">
```
and remove the `<br />` on the line below.

File: _pages/tourney-rules.html [hunk 3]
```diff
@@ -6,117 +6,109 @@
...
+                <li>Any <code>screenshare blockers</code> (firewalls, programs designed to mislead and block screenshare tools, etc.) will be considered an attempt to cloud the screenshare, and may be faced with a <b>Tournament Ban</b>.</li>
+            </ol>
+        </div>
+    </div>
+    <div class="pb-2 mt-4 mb-2 border-bottom">
+        <h2 class="page-heading">C. Infractions</h2>
+    </div>
+    <div class="row">
+        <div class="col-md-12">
+            <p><b>The infracions system works as follows:</b></p>
```
Comment (dentmaged) [inline _pages/tourney-rules.html:67]:
```suggestion
            <p><b>The infractions system works as follows:</b></p>
```

File: _pages/tourney-rules.html [hunk 4]
```diff
@@ -6,117 +6,109 @@
...
+        <div class="col-md-12">
+            <p><b>A match may be declared void if:</b></p>
+            <ol>
+                <li>Event servers and/or services have technical difficulties.</li>
+                <li>Unforeseen events cause severe impacts to the match.</li>
+                <li>A team or player is disqualified.</li>
+                <li>The match ended early for any reason.</li>
+                <li>The Heads of Events decide for any reason.</li>
+            </ol>
+            <p><b>A void match may be:</b></p>
```
Comment (dentmaged) [inline _pages/tourney-rules.html:98]:
```suggestion
            <p class="mb-0"><b>A void match may be:</b></p>
```

File: _pages/tourney-rules.html [hunk 5]
```diff
@@ -6,117 +6,109 @@
...
+                <li> Blacklisted players will not be allowed to register in future tournaments. You can try to appeal your punishment by contacting any <code>Head of Events</code>.</li>
+            </ul>
+        </div>
+    </div>
+    <div class="pb-2 mt-4 mb-2 border-bottom">
+        <h2 class="page-heading">E. Void Matches</h2>
+    </div>
+    <div class="row">
+        <div class="col-md-12">
+            <p><b>A match may be declared void if:</b></p>
```
Comment (dentmaged) [inline _pages/tourney-rules.html:90]:
```suggestion
            <p class="mb-0"><b>A match may be declared void if:</b></p>
```

File: _pages/tourney-rules.html [hunk 6]
```diff
@@ -6,117 +6,109 @@
...
+            </ol>
+        </div>
+    </div>
+    <div class="pb-2 mt-4 mb-2 border-bottom">
+        <h2 class="page-heading">D. Blacklist</h2>
+    </div>
+    <div class="row">
+        <div class="col-md-12">
+            <ul>
+                <li> Blacklisted players will not be allowed to register in future tournaments. You can try to appeal your punishment by contacting any <code>Head of Events</code>.</li>
```
Comment (dentmaged) [inline _pages/tourney-rules.html:81]:
```suggestion
                <li>Blacklisted players will not be allowed to register in future tournaments. You can try to appeal your punishment by contacting any <code>Head of Events</code>.</li>
```

File: _pages/tourney-rules.html [hunk 7]
```diff
@@ -6,117 +6,109 @@
...
+                <li><b>Tournament Ban:</b> This is the worst type of punishment a player may get. A team can receive up to three <code>Tournament Bans</code> before being disqualified from the tournament. This is reserved for extreme cases such as hacking or receiving up to <code>3 punishments</code>. For the first two Tournament Bans, the player will be disqualified from the tournament and the team will be given a loss for every map that player has played during that series. For the third one, the team will be disqualified from the tournament. <i>All players being issued a Tournament Ban will be <b>blacklisted</b> from future tournaments</i>.</li>
+            </ol>
+        </div>
+    </div>
+    <div class="pb-2 mt-4 mb-2 border-bottom">
+        <h2 class="page-heading">D. Blacklist</h2>
+    </div>
+    <div class="row">
+        <div class="col-md-12">
+            <ul>
```
Comment (dentmaged) [inline _pages/tourney-rules.html:80]:
```suggestion
            <ol>
```
I think this should be an ordered list like all the other rules on this page and the rules page.

**Review by Pablete1234 — APPROVED** [review]

**Review by Brottweiler — APPROVED** [review]

**Pull request merged**